tools/star_fusion_1.10.1_gen_reference.cwl

```diff
@@ -0,0 +1,58 @@
+cwlVersion: v1.2
+class: CommandLineTool
+id: star_fusion_1.10.1_gen_reference
+doc: "Generate star fusion reference. Recommend to run locally as tool will take at least 12 hours to run"
+requirements:
+  - class: ShellCommandRequirement
+  - class: DockerRequirement
+    dockerPull: 'pgc-images.sbgenomics.com/d3b-bixu/star:fusion-1.10.1'
+  - class: InlineJavascriptRequirement
+  - class: ResourceRequirement
+    coresMin: $(inputs.cores)
+    ramMin: ${ return inputs.ram * 1000 }
+  - class: InitialWorkDirRequirement
+    listing: [$(inputs.ctat_source)]
+
+baseCommand: [tar, -I pigz, -xvf]
+arguments:
+  - position: 2
+    shellQuote: false
+    valueFrom: >-
+      && /STAR-Fusion/ctat-genome-lib-builder/prep_genome_lib.pl
+      ${
+          var arg = "--dfam_db ";
+          for (var i = 0; i < inputs.dfam_db.length; i++) {
+              if (inputs.dfam_db[i].nameext=='.hmm'){
+                  arg += inputs.dfam_db[i].path;
+                  return arg;
+              }
+          }
+      }
+  - position: 4
+    shellQuote: false
+    valueFrom: >-
+      && tar -I pigz -cf $(inputs.output_dir).tar.gz $(inputs.output_dir)
+
+inputs:
+  ctat_source: { type: File, doc: "Resource file from https://data.broadinstitute.org/Trinity/CTAT_RESOURCE_LIB/", inputBinding: { position: 1} }
```

Comment:
So you put this tar in the working directory but then you're untaring the original path with the inputBinding? Wouldn't you want to untar what's in the working directory? Does the copy in the working directory even need to be there?